core/job.go

```diff
@@ -171,16 +172,29 @@ func startJob(r *http.Request) response {
 		logger.Error("Warning: Job with name %s has already been started for Train %d, Phase %d",
 			jobName, targetPhase.Train.ID, targetPhase.ID)
 
+		datadog.Incr("job.start", job.DatadogTags())
```

Comment:
Does anyone else use conductor?  Should we make the datadog requirement optional?

Reply:
Currently no one uses it. I'll add a flag to disable datadog.